ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -928,17 +928,17 @@ baseBounds ghcFlavor =
 -- | Common build dependencies.
 commonBuildDepends :: GhcFlavor -> [String]
 commonBuildDepends ghcFlavor =
-  [ "ghc-prim > 0.2 && < 0.8"
+  [ "ghc-prim > 0.2 && < 0.9"
```

Comment:
this looks like you’re changing the bounds for all releases. Is that intentional and correct? Can I build a GHC 8.10 ghc-lib with ghc-prim 0.8?

Reply:
i am and as we always have. i expect it to work. still working on things but will report back in a few minutes after i've run through the test suite on my fork to confirm.